src/main/java/zpi/algospace/solution/Cleaner.java

```diff
@@ -0,0 +1,21 @@
+package zpi.algospace.solution;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+@UtilityClass
+@Slf4j
+public class Cleaner {
+
+    public void cleanDirectory(List<String> filesToDelete) throws IOException {
```

Comment:
może czymś tą listę plików zastąpimy? taka luźna sugestia, możesz zostawić, a ja jak sie za twoj kod wezme to pomysle